datalad/cmd.py

```diff
@@ -524,8 +524,10 @@ def decode_if_not_None(x):
                         self._log_err(out[1], expected=expect_stderr)
 
                 if status not in [0, None]:
-                    msg = "Failed to run %r%s. Exit code=%d. out=%s err=%s" \
-                        % (cmd, " under %r" % (cwd or self.cwd), status, out[0], out[1])
+                    msg = "Failed to run %r%s. Exit code=%d.%s%s" \
+                        % (cmd, " under %r" % (cwd or self.cwd), status,
+                           "" if log_online else " out=" + out[0],
+                           "" if log_online else " err=" + out[1])
```

Comment:
according to failed test in https://travis-ci.org/datalad/datalad/jobs/399623568 `out[0]` could be None, so better to use `" out=%s" % out[0]` or alike (and the same for `err=`)